test/controllers/albums_controller_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+class AlbumsControllerTest < ActionController::TestCase
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_template :index
+  end
+
+   test "should get show" do
+    get :show, {id: albums(:no_escape) }
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, {id: albums(:no_escape) }
+    assert_response :success
+  end
+
+  test "should be able to update an album" do
+    update_params = {album:{name: albums(:no_escape).name, artist: albums(:no_escape).artist}, id: albums(:no_escape).id}
```

Comment:
While this is technically calling the `update` action and likely is changing data in the database, because we are using the same values that the model originally had (e.g. `albums(:no_escape).name`) we cannot actually assert that the post-update values are different.